Sources/SwiftDocCUtilities/Action/Actions/Convert/Indexer.swift

```diff
@@ -70,7 +70,7 @@ extension ConvertAction {
         
         /// Finalizes the index and writes it on disk.
         /// - Returns: Returns a list of problems if any were encountered during indexing.
-        func finalize(emitJSON: Bool, emitLMDB: Bool) -> [Problem] {
+        func finalize(emitJSON: Bool = true, emitLMDB: Bool) -> [Problem] {
```

Comment:
I agree it seems a little odd to provide the default here- I went ahead and removed that and made all the call-sites explicit.

> Maybe we should be removing this argument altogether (likewise in the NavigatorIndex builder).

I considered that but we actually have a few uses cases in testing where we want to build the in-memory model of the navigator tree but don't want to any kind of representation to disk.

I think our planned refactor of this will make this much clearer but for now leaving the parameter seems reasonable.

08a4ead0a75c8172d5b174c6b36560b389341431

Reply:
Makes sense.